virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -1433,39 +1455,81 @@ You can configure a MigrationPolicy on the MigrationPolicy details page.
 |Description
 
 |*Details* tab
-|Configure a MigrationPolicy by editing a form.
+|Configure a migration policy by editing a form.
 
 |*YAML* tab
-|Configure a MigrationPolicy by editing a YAML configuration file.
+|Configure a migration policy by editing a YAML configuration file.
 
 |*Actions* menu
 |Select *Edit* or *Delete*.
+|====
+=====
 
-|Name
-|MigrationPolicy name
+[id="migrationpolicy-details-details_virt-web-console-overview"]
+==== Details tab
+
+You configure a custom template on the *Details* tab.
 
+.*Details* tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
 |Description
-|MigrationPolicy description
 
-|Configurations
-|Click the edit icon to update the MigrationPolicy configurations.
+|*Name*
+|Migration policy name.
 
-|Bandwidth per migration
+|*Description*
+|Migration policy description.
+
+|*Configurations*
+|Click the edit icon to update the migration polic configurations.
+
+|*Bandwidth per migration*
 |Bandwidth request per migration. For unlimited bandwidth, set the value to `0`.
 
-|Auto converge
-|Auto converge policy
+|*Auto converge*
+|When auto converge is enabled, the platform compromises the performance and availability of the virtual machines to ensure that migration is successful.
```

Comment:
This use of "compromises" seems iffy to me because of the multiple meanings:

1. To arrive at a settlement by making concessions. <-- probably what was intended!
2. To reduce the quality, value, or degree of something, such as one's ideals. <-- how I read it at first